WWTMVC5/WWTMVC5.csproj

```diff
@@ -983,6 +989,7 @@
     <Content Include="WWTWeb\StarChart.aspx" />
     <Content Include="WWTWeb\test.aspx" />
     <Content Include="WWTWeb\testfailover.aspx" />
+    <Content Include="WWTWeb\testplate2.aspx" />
```

Comment:
Just as a general comment ... anything named along the lines of `testfoo` is, indeed, a one-off test and not essential. If it's easy to include, fine, but if gives you any trouble, just nuke it.